data/pages-content/humble-data.md

```diff
@@ -1,51 +1,58 @@
+Following the success of several Beginners Data Workshop at PyCon US, Ghana, Namibia, Africa and PyData Global, we are glad to have it again at EuroPython!
 
-👉 [APPLY HERE TO ATTEND THIS WORKSHOP](https://forms.gle/Y9thJjuAQAiFm8D8A) (application deadline is 4th July 2022) 👈
+👉🏽 [APPLY HERE TO ATTEND THIS WORKSHOP](https://forms.gle/jrF8uSb8w5tMhAyP8) (application deadline is 3rd July, 2023) 👈
 
 ## Learn Python for Data Science in this Beginners' Day Workshop ##
-Would you like to learn to code but don’t know where to start? Taking your first steps in programming can seem like an impossible task so we’ve decided to put on a workshop to show 50 beginners how it can be done and share our passion for the world of data science!
+Would you like to learn to code but don’t know where to start? Would you like to take a peek at Data Science but are not sure how? Taking first steps are very hard. That is why we have decided to host a workshop to teach beginners how to start with Python and Data Science. Join and let's share our passion for the world of data science!
 
 “We want our community to be more diverse: whoever you are, and whatever your background, we welcome you.” – Python Software Foundation’s Diversity Statement
 
-We invite those from under-represented groups in data to apply to join us on Tuesday 12th July 2022 for the EuroPython Beginners' Day - Humble Data Workshop. In this workshop you will learn the basics of programming in Python, as well as how to use tools such as Jupyter Notebook to analyse data.
+We invite those from under-represented groups to apply to join us on Monday 17th July 2023 for the EuroPython Beginners' Day - Humble Data Workshop. There, you will learn the basics of programming in Python, useful libraries and tools, such as Jupyter Notebook that help with data analysis.
 
-You will be learning in small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. This will all be facilitated online through the wonders of modern technology. You can expect plenty of exercises, quizzes and games, and inspiring talks from those who were once in your shoes.
+You will be divided into small groups, each with an assigned mentor to guide you through the workshop materials and answer your questions. You can expect plenty of exercises, quizzes and games, and inspiring talks from those who were once in your shoes.
 
 ## WHEN? ##
-Tuesday 12th July 2022 (time and room TBD).
+Monday, 17th July 2023 (time and room to be determined).
 
 ## WHERE? ##
-EuroPython (see [here](/where))
+EuroPython (see [here](/where)).
 
 ## WHO CAN JOIN? ##
-Anyone with a EuroPython [Conference Ticket or Combined Ticket](https://ep2022.europython.eu/tickets#ticket-types)
+Anyone with a EuroPython [Conference Ticket or Combined Ticket](https://ep2022.europython.eu/tickets#ticket-types).
```

Comment:
```suggestion
Anyone with a EuroPython  in-person [Conference , Tutorial or Combined Ticket](https://ep2023.europython.eu/tickets#ticket-types).
```

Reply:
@laisbsc this still needs to be changed: 2022-2023

Reply:
Awesome catch! TY.